### PR TITLE
Fix: Interface method spacing

### PR DESCRIPTION
**Pull request by dereuromark**
Fixes https://github.com/cakephp/cakephp/pull/13223/files#diff-f615e67b9f687da027b935bbf938fd8eR281

needs to be merged into cake4 one then as well.

Resolves
```php
interface AnotherInterface
{
    public function doSomethingElse();
    /**
     * @return void
     */
    public function doSomethingX();

    public function doSomethingY();
    public function doSomethingZ();
}
```

to
```php
interface AnotherInterface
{
    public function doSomethingElse();

    /**
     * @return void
     */
    public function doSomethingX();

    public function doSomethingY();

    public function doSomethingZ();
}
```

etc

### CODE REVIEW

File: CakePHP/Sniffs/Commenting/FunctionCommentSniff.php [hunk 1]
```diff
@@ -269,10 +268,9 @@ protected function processThrows(File $phpcsFile, $stackPtr, $commentStart)
                     $error = '@throws tag comment must end with a full stop';
                     $phpcsFile->addWarning($error, ($tag + 2), 'ThrowsNoFullStop');
                 }
-            }//end if
-        }//end foreach
-    }//end processThrows()
```
Comment (ADmad) [inline CakePHP/Sniffs/Commenting/FunctionCommentSniff.php]: Why have you removed all these really useful comments? :trollface:

Comment (dereuromark) [reply CakePHP/Sniffs/Commenting/FunctionCommentSniff.php]: I almost thought about adding also `;//end of statement` into the other lines for consistency.

**Comment by dereuromark** [comment]
Apparently full overwrite `@overwrite` is not supported on master branch yet, I reverted that part to {} partial use, even if not fully correct.

File: CakePHP/Sniffs/Commenting/FunctionCommentSniff.php (hunk 1 above)
Comment (markstory) [reply CakePHP/Sniffs/Commenting/FunctionCommentSniff.php]: I'm pretty sure these came for the ride when we pulled in from squizlabs.

**Comment by dereuromark** [comment]
I ran those on several repos and tested it on the fly, the cs tests are not that nice to work with afaik.
I wouldnt mind anyone adding them.

**Comment by dereuromark** [comment]
Can we merge this?

**Pull request merged**